tools/depends/native/binutils-native/Makefile

```diff
@@ -0,0 +1,38 @@
+include ../../Makefile.include
+PLATFORM=$(NATIVEPLATFORM)
+DEPS= ../../Makefile.include Makefile readelfOnly.patch
+
+# lib name, version
+LIBNAME=binutils
+VERSION=2.29
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.gz
+BASE_URL=https://ftp.gnu.org/gnu/binutils
```

Comment:
someone has to upload binutils-2.29.tar.gz to the mirrors and then this line can be removed